Code/CIPLabeler/Wrap/pyCIPLabelWrapTest.py

```diff
@@ -0,0 +1,64 @@
+import unittest
+
+from rdkit import Chem
+
+
+class TestCase(unittest.TestCase):
+
+  def testLabelAtomsList(self):
+    mol = Chem.MolFromSmiles("C[C@H](Cl)CC[C@H](Cl)C")
+    assert mol
+
+    atom1 = mol.GetAtomWithIdx(1)
+    atom5 = mol.GetAtomWithIdx(5)
+
+    assert atom1.HasProp("_CIPCode") == True
```

Comment:
please use the unittest methods instead of naked asserts
So this would become:
```suggestion
    self.assertTrue(atom1.HasProp("_CIPCode"))
```
this suggestion applies throughout this file